src/api/migrations/20190620192200-create-user.js

```diff
@@ -0,0 +1,70 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    username: {
```

Comment:
@nkpremices username and email should be unique.

Reply:
Noted @MemunaHaruna